### DISCUSSION TITLE
How to test gpu ?

### DISCUSSION THREAD

**Discussion by leadercxn** [OE4T/tegra-demo-distro#249] [General]
Hi OE4T developers, how can I test gpu function? I want to install jtop to watch GPU,but it occurred error when I `python3 setup.py  install` . And the yocto system couldn't `apt install nvidia-jetpack` , I don't know how to verify the gpu.
<img width="1276" alt="image" src="https://user-images.githubusercontent.com/43691182/218687005-62670d5b-a243-408f-8a78-e8ba4b462fcb.png">

**Comment by ichergui** [comment]
Hey @leadercxn 
There is different ways to test the iGPU 
You can simply run cuda sample applications (e.g deviceQuery) that are shipped into your yocto image.
BTW, we have the yocto [recipe for jtop](https://github.com/OE4T/meta-tegra-community/blob/master/recipes-devtools/python/python3-jetson-stats_3.1.2.bb) in our meta tegra community. Please try it and let me know how it goes

**Reply by leadercxn** [reply]
OK, I have found the way [here](https://docs.google.com/spreadsheets/d/1EWG5QZvtWdk74gz48D4xCbegK-bsqMFJ8XOsyGdUUeI/edit#gid=2136206154) , thanks~
